source/vocab/relations.ttl

```diff
@@ -189,6 +189,28 @@
     rdfs:subClassOf :Document;
     owl:equivalentClass sdo:MediaObject .
 
+:image a owl:ObjectProperty;
+    rdfs:label "image"@en, "bild"@sv;
+    rdfs:comment "En bild på resursen"@sv;
+    rdfs:domain :Resource;
+    rdfs:range :ImageObject;
+    rdfs:subPropertyOf :associatedMedia;
```

Comment:
```suggestion
    rdfs:subPropertyOf :isPrimaryTopicOf;
```
Since `associatedMedia` has a narrower domain (`:Endeavour`), we can't use it as a base property here. `isPrimaryTopicOf` works (though I'm not sure how useful it is; perhaps just skip any base for this).